src/app/legacy/components/OptimoPromos/MediaIndicator/index.styles.jsx

```diff
@@ -10,4 +11,9 @@ export const MediaIndicatorAlignment = styled.div`
   display: flex;
   align-items: center;
   height: 100%;
+  > svg {
+    @media screen and (-ms-high-contrast: active) {
+      fill: ${C_WHITE};
```

Comment:
It might be better to use `fill: CanvasText`, not actually sure if it works, reading [this](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#system_colors) it looks like it will automatically set the colour to whatever normal text colour changes to when high contrast mode is enabled. The definitions for these System Colours are [here](https://www.w3.org/TR/css-color-4/#valdef-system-color-canvastext)